platform/linuxbsd/wayland/wayland_thread.cpp

```diff
@@ -5375,9 +5375,19 @@ void WaylandThread::destroy() {
 			zwp_tablet_tool_v2_destroy(tool);
 		}
 
+		zwp_text_input_v3_destroy(ss->wp_text_input);
```

Comment:
Probably should have `null` check, even if it's expected to be always initialized:


```suggestion
	if (ss->wp_text_input) {
		zwp_text_input_v3_destroy(ss->wp_text_input);
	}
```